Q/QuantumIterativeLearningControl/Package.toml

```diff
@@ -1,3 +1,3 @@
 name = "QuantumIterativeLearningControl"
 uuid = "b2c2a83c-20aa-4d51-8862-bb551092e619"
-repo = "https://github.com/aarontrowbridge/QuantumIterativeLearningControl.jl.git"
+repo = "https://github.com/harmoniqs/QuantumIterativeLearningControl.jl.git"
```

Comment:
This repo doesn't exist (or isn't public?) anymore

Reply:
You're right, we haven't got it back to a publishable state. Should I just revert this change from the PR?

Reply:
Yes, please

Reply:
Reverted